velox/docs/functions/presto/conversion.rst

```diff
@@ -1271,3 +1271,20 @@ Miscellaneous
         SELECT typeof(123); -- integer
         SELECT typeof(1.5); -- double
         SELECT typeof(array[1,2,3]); -- array(integer)
+
+From interval year month
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
We need to move this description to the section `Cast to Integral Types` at https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/presto/conversion.rst#cast-to-integral-types.